optimize/cmaes.go

```diff
@@ -0,0 +1,383 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package optimize
+
+import (
+	"math"
+	"math/rand"
+	"sort"
+	"sync"
+
+	"gonum.org/v1/gonum/stat/distmv"
+
```

Comment:
Delete the blank lines?

Reply:
Huh. I don't know how that happened... I've only been using go imports.

Reply:
Fixed.

Reply:
I think that happens with successive goimports additions that are not stdlib.